internal/cmd/providers.go

```diff
@@ -36,100 +37,166 @@ func providersCmd() *cobra.Command {
 
 func createProviderCmd() *cobra.Command {
 	var (
-		hostname       string
-		clusterAddress string
+		fromContext   bool
+		clusterCA     string
+		clusterAddr   string
+		SAToken       string
+		namespace     string
+		storageClass  string
+		sshEnabled    bool
+		environmentSA string
+		orgAllowlist  []string
 	)
 	cmd := &cobra.Command{
-		Use:   "create [name] --hostname=[hostname] --cluster-address=[clusterAddress]",
+		Use:   "create [name]",
 		Args:  xcobra.ExactArgs(1),
 		Short: "create a new workspace provider.",
 		Long:  "Create a new Coder workspace provider.",
```

Comment:
```suggestion
		Long:  `Create a new Coder workspace provider.

Creates a new Kubernetes workspace provider with the specified details. No resources are created or validated on the cluster.`,
```

Reply:
Can we add a sentence about workspace providers here? just helps in case people don't already know what they are. Something like: "Create a workspace provider and connect it with your Coder installation. Workspace providers enable Coder to dynamically provision and decommission(?) developer workspaces in any Kubernetes cluster."

Not sure if "decommission" is the right word to use here...